source/Cosmos.System2_Plugs/System/ConsoleImpl.cs

```diff
@@ -13,7 +13,9 @@ public static class ConsoleImpl
         private static ConsoleColor mForeground = ConsoleColor.White;
         private static ConsoleColor mBackground = ConsoleColor.Black;
         private static Encoding ConsoleInputEncoding = Encoding.ASCII;
-        private static Encoding ConsoleOutputEncoding = Encoding.ASCII;
+        private static Encoding ConsoleOutputEncoding = Encoding.ASCII;
+        private static Core.Processing.Mutex mConsoleGateRead = new Core.Processing.Mutex();
+        private static Core.Processing.Mutex mConsoleGateWrite = new Core.Processing.Mutex();

```

Comment:
Please specify Cosmos.Core.Processing with an using rather than this

Reply:
Right. will fix in a moment